pkg/docker/index.html

```diff
@@ -496,6 +537,28 @@ <h4 class="modal-title">Run Image</h4>
               </td>
             </tr>
             <tr>
+              <td><label class="control-label" for="mount-volumes" translatable="yes">Volumes</label></td>
+              <td colspan="4">
+                <label>
+                  <input type="checkbox" name="mount-volumes" id="mount-volumes" checked="false"/>
+                  Mount container volumes
+                </label>
+                <div id="select-mounted-volumes" class="containers-run-volumemount containers-run-inline">
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td><label class="control-label" for="claim-envvars" translatable="yes">Env Vars</label></td>
+              <td colspan="4">
+                <label>
+                  <input type="checkbox" name="claim-envvars" id="claim-envvars" checked="false"/>
+                  Claim container environment variables
```

Comment:
This line would be clearer if it said 'Set environment variables'. In addition the text here should be inside a ```<span translatable="yes">```